packages/fluentui/react-northstar/src/components/Card/Card.tsx

```diff
@@ -91,19 +99,25 @@ const Card: React.FC<WithAsProp<CardProps>> &
     fluid,
     onClick,
     disabled,
+    selected,
   } = props;
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(Card.handledProps, props);
   const getA11yProps = useAccessibility(props.accessibility, {
     debugName: Card.displayName,
     actionHandlers: {
       performClick: e => {
+        // prevent Spacebar from scrolling
+        e.preventDefault();
```

Comment:
Can this be executed in a context of a submission on Enter? Like submitting a form POST? If so, this will prevent it.

Reply:
I don't think cards would have submission elements inside, @jurokapsiar wdyt?

Reply:
I would only expect space to select/unselect, not enter - similar to checkbox or selectable tree items.

Reply:
Fixed, now selectable card has a separate behavior that triggers preformClick only on Spacebar